
Yuck. Ok, let's patch the self extracting file slightly ;-).

  - Rene Rebe <rene@exactcode.de>

--- mtxdrivers-rh9.0-0.3.0.run	2005-01-27 18:24:57.560943152 +0100
+++ mtxdrivers-rh9.0-0.3.0.run	2005-01-27 18:41:02.832199616 +0100
@@ -5,7 +5,7 @@
 TMPROOT=${TMPDIR:=/tmp}
 
 label="Matrox Parhelia Driver"
-script="./install.sh"
+script="true"
 scriptargs=""
 echo -n "Please, enter the directory to extract the files [$HOME/] "
 read str_installdir
@@ -89,7 +89,7 @@
     MD5_PATH=${MD5_PATH:-`which md5 2>/dev/null || type md5 2>/dev/null`}
     PATH=$OLD_PATH
     MS_Printf "Verifying archive integrity..."
-    offset=`head -345 "$1" | wc -c | tr -d " "`
+    offset=`head -n 345 "$1" | wc -c | tr -d " "`
     verbose=$2
     i=1
     for s in $filesizes
@@ -185,7 +185,7 @@
 	;;
     --list)
 	echo Target directory: $targetdir
-	offset=`head -345 "$0" | wc -c | tr -d " "`
+	offset=`head -n 345 "$0" | wc -c | tr -d " "`
 	for s in $filesizes
 	do
 	    MS_dd "$0" $offset $s | eval "gzip -cd" | UnTAR t
@@ -299,7 +299,7 @@
 if test x$SETUP_NOCHECK != x1; then
     MS_Check "$0"
 fi
-offset=`head -345 "$0" | wc -c | tr -d " "`
+offset=`head -n 345 "$0" | wc -c | tr -d " "`
 
 MS_Printf "Uncompressing $label"
 res=3
